src/test/java/net/snowflake/client/jdbc/StatementLatestIT.java

```diff
@@ -225,4 +226,26 @@ public void testPreparedStatementLogging() throws SQLException {
       }
     }
   }
+
+  @Test // SNOW-647217
+  public void testSchemaWith255Characters() throws SQLException {
+    String schemaName = generateRandomString(255);
+    try (Connection con = getConnection()) {
+      try (Statement stmt = con.createStatement()) {
+        stmt.execute("create schema " + schemaName);
+        stmt.execute("use schema " + schemaName);
+        stmt.execute("drop schema " + schemaName);
```

Comment:
It would be nice to have indication than then/expect is "no exception thrown".

Reply:
when exception is thrown then junit fail the test
we can set `@Test(expected = None.class)` but it's default

Reply:
Yes, I just wanted to make actual expectations more descriptive. But I see that `assertDoesNotThrow` was introduced in Junit 5. So if you don't have some good idea, you can close this comment.